md/customize-living-application-layout.md

```diff
@@ -1,11 +1,11 @@
-# Customize living application layout
+# Customize the living application layout
 
 ::: info
 **Note:** For Performance, Efficiency, and Teamwork editions only.
 :::
 
 ::: warning
-Since 7.3 we made some changes on the default layout. You will have to remove `layout.css` from the page asset to make the following instructions relevant.
+Since Bonita BPM 7.3, we have made some changes in the default layout. You will have to remove `layout.css` from the page asset to make the following instructions relevant.
 :::
```

Comment:
So what's the option if we want to avoid the bug and still customize the layout?
It seems that it's either/or but not both.